sycl/include/syclcompat/atomic.hpp

```diff
@@ -151,42 +88,12 @@ template <typename T,
               sycl::access::address_space::global_space,
           sycl::memory_order memoryOrder = sycl::memory_order::relaxed,
           sycl::memory_scope memoryScope = sycl::memory_scope::device>
```

Comment:
Should T be the last template parameter? This applies in multiple places

Reply:
Yes :facepalm: thanks!